src/argparse.lua

```diff
@@ -511,17 +513,29 @@ function Argument:_get_argname(narg)
    end
 end
 
+function Argument:_get_choices_list()
+   return "{" .. table.concat(self._choices, ",") .. "}"
+end
+
 function Argument:_get_default_argname()
-   return "<" .. self._name .. ">"
+   if self._choices then
+      return self:_get_choices_list()
+   else
+      return "<" .. self._name .. ">"
+   end
 end
 
 function Option:_get_default_argname()
-   return "<" .. self:_get_default_target() .. ">"
+   if self._choices then
```

Comment:
Do we really want this case? What if you have a default choice?

Reply:
`_get_default_target` just returns the first long alias, without the `--`. The argname for e.g. `-f --format` would normally be `<format>`. This makes it show the choices instead of there are any. It can still be overridden with `:argname`.